src/unity/schemas/RelatedAccount.yml

```diff
@@ -2,14 +2,23 @@ description: Subset of Account data related to the organization
 type: object
 properties:
   id:
-    type: string
-    description: account id
-  email:
```

Comment:
we still need the email here to display the organization's account email in the operator/orgs page, unless that's accessible via another property like the billing contact? But if that's the case then we'd need to update the billingContact in the RelatedAccount type to be required rather than optional. Thoughts?

Reply:
Email is currently coming in with the billing contact. The reason I made billing contact not required is because both `operator/orgs` and `operator/orgs/:orgId` share the related account schema, but the latter doesn't currently return billing contact. I suppose we could keep it as required and it can just be an empty object for the latter?

Reply:
making the billingContact required sounds good to me

Reply:
Okay. I will add it back in. 😄

Reply:
What impact does marking a field `required` have on the client code? My initial assumption was that it's really more for POST/PUT/PATCH requests and not GETs. Are there more semantics to it than that?

I don't actually care one way or the other whether we mark billing contact as required, but I don't want to have to return it from the specific org GET request if it isn't needed there because that's just increasing payload size (and database work) unnecessarily.  We know it's needed from the GET all orgs request, and it is returned there as needed.

So either way, the system isn't going to break.

So what's the best way to specify this distinction in the swagger without having to create another entity type? I have very limited experience with swagger, so I'm not sure what the right answer is here.

If we can mark the field as required without returning it from the one request, I'm fine with that. If we can mark the field as optional while still returning it from the other request, I'm fine with that as well. What does the client need from the swagger spec in order for this to work?

Reply:
It's just to enforce the contract between the expected output for the UI. If having an email field for an associated account on the operator page is an optional field, then we can make the billing contact or email optional and go from there. I don't have much context on the usage of the operator page so it's hard for me to make a judgment call on what should exist and what shouldn't. Based on the UI that was ported over, the expectation was that an associated account email would exist for the org/id path, but if that's not the case then we can just mark it as option and leave it blank if none exists

Reply:
Final decision: `zuoraAccountId` and `deletable` have been removed from `RelatedAccount` schema; they may still be sent, but aren't part of the swagger contract. `billingContact` has been made optional since it is only needed for one of the routes that utilize the schema, as discussed/approved by @Asalem1 .